src/asphalt/core/cli.py

```diff
@@ -71,6 +83,26 @@ def run(configfile, unsafe: bool, loop: Optional[str], service: Optional[str]) -
     if loop:
         config["event_loop_policy"] = loop
 
+    # Read the configuration from the CLI
+    for cli_conf in set:
+        if "=" not in cli_conf:
+            click.echo(f"Configuration must be set with '=', got: {cli_conf}")
+            raise click.Abort()
+
+        key, value = cli_conf.split("=", 1)
```

Comment:
`value` is always a string, while parsing YAML files automatically casts to e.g. numbers when appropriate.

Reply:
Actually this is fine, casting is done by Asphalt.